File: src/windows/do_Http/extdefine/Do_Http_MAbstract.cs
using doCore.Object;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace do_Http.extdefine
{
    public abstract class do_Http_MAbstract : doMultitonModule
    {
        protected do_Http_MAbstract()
            : base()
        {

        }
        /// <summary>
        /// 初始化
        /// </summary>
        public override void OnInit()
        {
            base.OnInit();
            //注册属性
        }
    }
}

File: src/android/doExt_Do_Http/src/extdefine/Do_Http_MAbstract.java
package extdefine;

import core.object.DoMultitonModule;
import core.object.DoProperty;
import core.object.DoProperty.PropertyDataType;


public abstract class Do_Http_MAbstract extends DoMultitonModule{

	protected Do_Http_MAbstract() throws Exception {
		super();
	}
	
	/**
	 * 初始化
	 */
	@Override
	public void onInit() throws Exception{
        super.onInit();
        //注册属性
		this.registProperty(new DoProperty("body", PropertyDataType.String, "", false));
		this.registProperty(new DoProperty("contentType", PropertyDataType.String, "text/html", false));
		this.registProperty(new DoProperty("method", PropertyDataType.String, "get", false));
		this.registProperty(new DoProperty("timeout", PropertyDataType.Number, "5000", false));
		this.registProperty(new DoProperty("url", PropertyDataType.String, "", false));
	}
}